tools/pd-heartbeat-bench/main.go

```diff
@@ -293,24 +294,37 @@ func (rs *Regions) update(cfg *config.Config, options *config.Options, indexes [
 			Put: uint64(queryUnit * rand.Float64()),
 		}
 		updatedRegionsMap[i] = region
+		updatedStatisticsMap[i] = region
 	}
 	// update interval
 	for _, region := range rs.regions {
 		region.Interval.StartTimestamp = region.Interval.EndTimestamp
 		region.Interval.EndTimestamp = region.Interval.StartTimestamp + regionReportInterval
 	}
-	for _, region := range updatedRegionsMap {
+	for _, i := range reportRegions {
+		region := rs.regions[i]
+		if r, exist := updatedRegionsMap[i]; exist {
```

Comment:
What do `L306-L308`  mean

Reply:
replace by the region which is updated above